.github/workflows/circle_artifacts.yml

```diff
@@ -2,7 +2,9 @@ on: [status]
 jobs:
   circleci_artifacts_redirector_job:
     runs-on: ubuntu-22.04
-    if: "github.repository == 'scipy/scipy' && !contains(github.event.head_commit.message, '[skip circle]')  && github.event.context == 'ci/circleci: build_docs'"
+    if: >
+      github.repository == 'scipy/scipy'
+      && github.event.context == 'ci/circleci: build_docs'
```

Comment:
As far as I can tell, the behaviour is identical whether `&& !contains(github.event.head_commit.message, '[skip circle]')` is included or not.